docs/topics/braintree.rst

```diff
@@ -261,3 +260,27 @@ Some information about the subscripton is stored in solitude.
     :query paymethod__braintree_buyer: the primary key of the braintree buyer.
     :query paymethod__braintree_buyer__buyer: the primary key of the buyer.
     :query seller_product: the primary key of the product.
+
```

Comment:
hmm, I wonder if you should add a rST comment here mentioning that the ``payments-service`` docs will link to this anchor per https://github.com/mozilla/payments-service/pull/21/files#diff-7d64b15bbaac9444c9da295924bf42bfR188